scripts/README.md

```diff
@@ -0,0 +1,4 @@
+# Scripts
+
```

Comment:
What do you think about adding a bit of context about what kind of scripts live in here? I see there's only one file at the moment so my assumption is that it's helpful scripts for testing but I'm not completely sure

Reply:
The check-translations script was put here very early on as it was the only shared place we had at the time. This script should be pulled out into a separate repository so it can be reused by other teams - #407 covers that

Apart from that, I can't think of any scripts that this package should need that are unique - I'd expect them all to be configuration to other systems (e.g. webpack) or something managed by the FEC team.